OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -382,4 +382,9 @@ public interface ITargetableCells
 	{
 		IEnumerable<Pair<CPos, SubCell>> TargetableCells();
 	}
+
+	public interface IGiveBuildableArea
```

Comment:
This interface doesn't give us anything, so please just reference `GivesBuildableArea` directly from `BuildingInfo`.